README.md

```diff
@@ -3,7 +3,7 @@
 ## Overview
 This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project.  By using the [OpenAPI-spec](https://www.openapis.org/) from a remote server, you can easily generate an API client.
 
-- API version: 2023-04-10
+- API version: development
```

Comment:
Do you know what caused this change? It seems unexpected and meaningful.

Reply:
This is deliberate, because the Cloud 2.0 API changes are still in development. I'm planning not to merge this into `main`, but into the `api-cloud-20` branch. I'm just diff'ing with `main` because the diff with `api-cloud-20` is a mess, because branches have diverged.